crates/manifest/Cargo.toml

```diff
@@ -14,6 +14,7 @@ thiserror = "1"
 terminal = { path = "../terminal" }
 toml = { version = "0.8.0", features = ["preserve_order"] }
 url = "2.4.1"
+wasm-pkg-loader = { workspace = true }
```

Comment:
I am distinctly wary of taking a dependency on the package loader in the manifest crate.

Reply:
Could use https://docs.rs/wasm-pkg-common/latest/wasm_pkg_common/ instead; the relevant types are re-exported from that by wasm-pkg-loader.

Reply:
Ooh, that looks perfect

Reply:
How distinctly wary of [wasm-pkg-common](https://docs.rs/wasm-pkg-common/0.4.1/wasm_pkg_common/all.html) would you be?

Reply:
Oh dangit

Reply:
🥷